sdk/containerregistry/azure-containerregistry/azure/containerregistry/aio/_async_base_client.py

```diff
@@ -32,8 +31,7 @@ class ContainerRegistryBaseClient(object):
 
     def __init__(
         self, endpoint: str, credential: "AsyncTokenCredential", **kwargs
-    ):  # pylint:disable=client-method-missing-type-annotations
-        # type: (str, TokenCredential, Dict[str, Any]) -> None
+    ):  # pylint: disable=client-method-missing-type-annotations
```

Comment:
seems like we could delete this pylint ignore. Not sure if it's yelling about the return type, though